rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -1411,6 +1419,13 @@ void main_window::CreateConnects()
 		}
 	});
 
+#if defined(_WIN32) || defined(__linux__)
+	connect(ui->updateAct, &QAction::triggered, [=]()
+	{
+		updater.check_for_updates(false, this);
```

Comment:
Check for Emu.IsStopped() and spawn a messagebox that tells you to stop the emulation first